src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -136,8 +136,13 @@ private LiKafkaConsumerImpl(LiKafkaConsumerConfig configs,
     DeliveredMessageOffsetTracker messageOffsetTracker = new DeliveredMessageOffsetTracker(maxTrackedMessagesPerPartition);
 
     // Instantiate auditor if needed.
-    Auditor<K, V> auditor = consumerAuditor != null ? consumerAuditor :
-        configs.getConfiguredInstance(LiKafkaConsumerConfig.AUDITOR_CLASS_CONFIG, Auditor.class);
+    Auditor<K, V> auditor;
```

Comment:
Yep, it wasn't clear from the doc, unfortunately. But make sense.
I've fixed the documentation.